File: dsaWithJava/tree/avltree/Node.java
package tree.avltree;

public class Node {
    int data, height;
    Node left, right;

    Node() {

    }

    Node(int data) {
        this.data = data;
        this.height = 1;
        this.left = this.right = null;
    }

    Node(int data, Node left, Node right) {
        this.data = data;
        this.height = 1;
        this.left = left;
        this.right = right;
    }
}

File: Python/tree/avl_tree/node.py
class Node:
    def __init__(self) -> None:
        pass

    def __init__(self, data):
        self.data = data
        self.height = 1
        self.left = self.right = None

    def __init__(self, data, left=None, right=None):
        self.data = data
        self.height = 1
        self.left = left
        self.right = right
